modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java

```diff
@@ -68,6 +68,30 @@ public void tearDown() throws Exception {
 		super.tearDown();
 	}
 
+	@Override
+	@Test
+	public void testGetListTypeDefinitionByExternalReferenceCodeListTypeEntriesPageWithSortInteger()
+		throws Exception {
+
+		testGetListTypeDefinitionByExternalReferenceCodeListTypeEntriesPageWithSort(
+			EntityField.Type.INTEGER,
+			(entityField, listTypeEntry1, listTypeEntry2) -> {
+				if (BaseListTypeDefinitionResourceTestCase.BeanTestUtil.
```

Comment:
Hey @pedro-oliveira446,
where did you see this pattern?

Could we use the BeanTestUtil.hasProperty() ? Why do we need to use BaseListTypeDefinitionResourceTestCase.BeanTestUtil?

Reply:
Hi @marcelabc,
I think it was either SF or AFCB that replaced this automatically, but now I have changed it.